pkg/util/utils.go

```diff
@@ -0,0 +1,28 @@
+// Copyright © 2021 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
```

Comment:
personally I'd avoid util packages (https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common), tho I don't feel very strongly about it so feel very free not to fix it in this PR / at all - just a suggestion :)

Reply:
works for me